From 5bb16a57ac8873d5a8fdcbc91a85a68733e7c310 Mon Sep 17 00:00:00 2001
From: dkl <dkl@rock-chips.com>
Date: Mon, 20 Jan 2014 19:36:52 +0800
Subject: [PATCH] clock: rockchip: ignore unused clocks temporarily.

Include "clk_ignore_unused" in the bootargs to the kernel.
---
 arch/arm/boot/dts/rk3188-tb.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk3188-tb.dts b/arch/arm/boot/dts/rk3188-tb.dts
index 4db2927e903e..c873643cdcad 100644
--- a/arch/arm/boot/dts/rk3188-tb.dts
+++ b/arch/arm/boot/dts/rk3188-tb.dts
@@ -11,7 +11,7 @@
 	};
 
 	chosen {
-		bootargs = "";
+		bootargs = "clk_ignore_unused";
 	};
 
 	fiq-debugger {
-- 
2.35.3

